browser/src/control/Control.QuickFindPanel.ts

```diff
@@ -33,13 +33,68 @@ class QuickFindPanel extends SidebarBase {
 		this.map.off('quickfind', this.onQuickFind, this);
 	}
 
+	onJSUpdate(e: FireEvent): void {
+		var data = e.data;
+
+		if (data.jsontype !== this.type) return;
+		if (!this.container) return;
+		if (!this.builder) return;
```

Comment:
we could just call an original method from the base class to not copy the code
but fine for now, can be followup cleanup PR